core/external_search.py

```diff
@@ -2020,6 +2029,36 @@ class Operators(Values):
         "open_access": "licensepools.open_access",
         "available": "licensepools.available",
         "classification": "classifications.term",
+        "data_source": "licensepools.data_source_id",
+    }
+
+    class ValueTransforms:
+        @staticmethod
```

Comment:
Please document the expected data type and format if applicable for value parameter and specify the return types in the signature.